sys/conf/options.arm64

```diff
@@ -3,7 +3,6 @@
 ARM64				opt_global.h
 INTRNG				opt_global.h
 SOCDEV_PA			opt_global.h
-SOCDEV_VA			opt_global.h
```

Comment:
Why's this gone?

Reply:
Gone as a result of 43a560d97d836cca72311b473566d4e22387a821 cherry picked from upstream. If I understand correctly, instead of using a fixed address it will pick some spare space at boot to map to the `SOCDEV_PA` address. This is left as a todo for Morello support as we are not using the `SOCDEV_PA` option.

Reply:
:+1: